ncl/ncl_entries/days_in_month.ipynb

```diff
@@ -0,0 +1,194 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7bfc78baeee6cef4",
+   "metadata": {},
+   "source": [
+    "## Using `cftime`\n",
+    "[`cftime`](https://unidata.github.io/cftime/index.html) supports all of the [CF Convention calendars](https://cfconventions.org/cf-conventions/cf-conventions#calendar). Here's how you can use `cftime` to calculate the number of days in a month."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "f762cbca2cf59dfa",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-06-02T23:36:55.219575Z",
+     "start_time": "2024-06-02T23:36:55.126580Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "standard calendar: \t\t\t\t month 2 in 1500 has 29 days\n",
+      "proleptic Gregorian calendar: \t month 2 in 1500 has 28 days\n",
+      "Julian calendar: \t\t\t\t month 2 in 1500 has 29 days\n",
+      "360 day calendar: \t\t\t\t month 2 in 1500 has 30 days\n"
+     ]
+    }
+   ],
+   "source": [
+    "import cftime\n",
+    "\n",
+    "day = 1\n",
+    "month = 2\n",
+    "year = 1500\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='standard').daysinmonth\n",
+    "print(f\"standard calendar: \\t\\t\\t\\t month {month} in {year} has {days} days\")\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='proleptic_gregorian').daysinmonth\n",
+    "print(f\"proleptic Gregorian calendar: \\t month {month} in {year} has {days} days\")\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='julian').daysinmonth\n",
+    "print(f\"Julian calendar: \\t\\t\\t\\t month {month} in {year} has {days} days\")\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='360_day').daysinmonth\n",
+    "print(f\"360 day calendar: \\t\\t\\t\\t month {month} in {year} has {days} days\")"
+   ]
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "## Differences between `cftime` and NCL's `days_in_month`\n",
+    "\n",
+    "### Calendars\n",
+    "The calendars available in `cftime` and NCL's `days_in_month`, while both based on the CF Conventions calendars, are slightly different.\n",
+    "\n",
+    "- `cftime` does support using `\"none\"` as a calendar type, but not for this particular function. \n",
+    "- NCL's `days_in_month` does not support the `proleptic_gregorian` calendar.\n",
+    "\n",
+    "### Numerical Differences\n",
+    "Additionally, the `\"standard\"` and `\"gregorian\"` calendars seem to disagree for some historical dates.\n",
+    "\n",
+    "### Input type\n",
+    "Notably, `cftime.datetime().daysinmonth` only takes in integer values for `year`, `month`, and `day`, while NCL's `days_in_month` can take in multidimensional integer arrays, given that the `year` and `month` arrays have the same dimensions."
```

Comment:
Isn't this a bit more nuanced since daysinmonth is a method that acts on cftime.datetime objects?  It's a different usage pattern, but from reading this I think I would interpret it as more limited than it is in practice.

Reply:
Late night review got me there.  

Maybe we keep it simple for now?

Reply:
I'll do a light re-word

Reply:
I've updated it in the last batch of commits to say:
> Notably, using `cftime` to calculate the number of days in a month works by getting a `daysinmonth` value from a `cftime.datetime` object, which means that the calculations for days in a month have to be collected through each date individually, while NCL's `days_in_month` can take in multidimensional integer arrays, given that the `year` and `month` arrays have the same dimensions.